lib/Extra/Tree.elm

```diff
@@ -0,0 +1,8 @@
+module Extra.Tree exposing (hasChildren)
+
+import Tree exposing (Tree)
+
+
+hasChildren : Tree a -> Bool
+hasChildren x =
+    List.length (Tree.children x) /= 0
```

Comment:
wouldn't this be more performant, while having the same level of clarity
```
case Tree.children x of
    [] -> False
    _ -> True
```

Reply:
Resolved without change?